components/n-ui/tooltip/index.js

```diff
@@ -78,7 +78,9 @@ export class NToolTip{
 	}
 
 	position (){
-		let left = (this.el.offsetLeft - (this.tooltip.offsetWidth / 2));
+		const elRect = this.el.getClientRects()[0];
+		const parentRect = this.el.parentNode.getClientRects()[0];
```

Comment:
I'm assuming this element should be `position: relative` but in case it isn't, should this be `offsetParent`?